quadcore/config/__init__.py

```diff
@@ -0,0 +1,36 @@
+from dotenv import load_dotenv, find_dotenv
+import os
+
+load_dotenv(find_dotenv())
+
+class Config:
+    # Configuration for redis
+    redis = {
+        "host": os.environ.get("REDIS_HOST"),
+        "id": os.environ.get("REDIS_ID"),
+        "password": os.environ.get("REDIS_PASSWORD"),
+        "port": os.environ.get("REDIS_PORT"),
+        "db": os.environ.get("REDIS_DB_NUMBER")
+    }
+
+    # RSS feed links
+    rss_links = {
+        "The Verge": "https://www.theverge.com/rss/index.xml",
+        "Engadget": "https://www.engadget.com/rss.xml",
+        "LifeHacker": "https://lifehacker.com/rss",
+        "Wired": "https://www.wired.com/feed/rss"
+    }
+
+    # Positional ordered: name, publish, content, author
+    rss_props = {
```

Comment:
what's a prop?
use full words like rss_properties

Reply:
I renamed it from `rss_properties` to `rss_props` because I thought it can be **too long** on real codes because we have character limitation per line (80 chars). I'll rename it with a short and meaningful name. 😄